src/common-components/BaseComponent.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Responsive from 'react-responsive';
+
+import SmallScreenLayout from './SmallScreenLayout';
+import MediumScreenLayout from './MediumScreenLayout';
+import LargeScreenLayout from './LargeScreenLayout';
+
+const BaseComponent = ({ children }) => (
+  <>
+    <Responsive maxWidth={767}>
```

Comment:
I guess we can use Paragon's responsive components instead https://paragon-edx.netlify.app/components/responsive/